File: CoreJava/game-tic-tac-toe-facade-app/src/com/techlab/game/Player.java
package com.techlab.game;

public class Player {

	private String playerName;
	private Mark mark;

	public Player(String playerName, Mark mark) {
		this.playerName = playerName;
		this.mark = mark;
	}

	public Mark getMark() {
		return mark;
	}

	public String getPlayerName() {
		return playerName;
	}

}

File: C#Folder/TicTacToeGameSolution/TicTacToeGameLib/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TicTacToeGameLib
{
    public class Player
    {
		private String playerName;
		private Mark mark;

		public Player(String playerName, Mark mark)
		{
			this.playerName = playerName; 
			this.mark = mark;
		}

		public Mark Mark { get => mark; }

		public String PlayerName { get => playerName; }
	}
}
